### ISSUE TITLE
When will we see an update to the conversation api that rolled out july 11?

### ISSUE THREAD

**Issue by dennisnotojr** [watson-developer-cloud/ios-sdk#372]
Trying to use the iOS sdk for the production version for conversation api.  The Message response def needs updating as well as the url.  Tried doing it myself but have run into  JSON serialization errors on the conversation.message call.  Hope to see it soon. Thanks, Den

**Comment by dennisnotojr** [comment]
Here's the error I'm seeing  .Error Domain=com.alamofire.error Code=-6004 "Data could not be serialized. Failed to parse JSON response. Unexpected JSON value (["text": Who is your creator]) was found that is not convertible to the type String." UserInfo={NSLocalizedFailureReason=Data could not be serialized. Failed to parse JSON response. Unexpected JSON value (["text": Who is your creator]) was found that is not convertible to the type String.}

**Comment by dennisnotojr** [comment]
Here is the file I changed. Plus the change to the URL in in Conversation.swift.  Thought that was all I needed to do. Apparently I messed some where. Named the swift file .txt so it could be loaded in. Weird that its complaining about the serializing the text input on the request because I didn't change that part. Thanks, Den
[MessageResponse copy.txt](https://github.com/watson-developer-cloud/ios-sdk/files/362777/MessageResponse.copy.txt)
